python-auto-metadata-collection/main_sklearn.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+from ai4hf_passport_sklearn import SKLearnMetadataCollectionAPI
+from ai4hf_passport_models import LearningStage, EvaluationMeasure, Model, LearningStageType, EvaluationMeasureType
+# Example usage of sklearn library
+dataset = pd.read_csv('Social_Network_Ads.csv')
+X = dataset.iloc[:, :-1].values
+y = dataset.iloc[:, -1].values
+
+from sklearn.model_selection import train_test_split
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.25, random_state = 0)
+
+from sklearn.preprocessing import StandardScaler
+sc = StandardScaler()
+X_train = sc.fit_transform(X_train)
+X_test = sc.transform(X_test)
+
+from sklearn.tree import DecisionTreeClassifier
+classifier = DecisionTreeClassifier(criterion = 'entropy', random_state = 0)
+classifier.fit(X_train, y_train)
+
+# Construct an api client for interacting with AI4HF passport server
+api_client = SKLearnMetadataCollectionAPI(
+        passport_server_url="http://localhost:80/ai4hf/passport/api",
+        study_id="1",
+        organization_id="1",
+        username="data_scientist",
+        password="data_scientist"
+    )
+
+# Provide learning stages
+learning_stages = [
+    LearningStage(learningStageType = LearningStageType.TRAINING,
+                  datasetPercentage = 70),
+    LearningStage(learningStageType = LearningStageType.TEST,
+                  datasetPercentage = 30)
```

Comment:
At line 10, we have test_size = 0.25. Let's make sure that the examples make sense for the readers to prevent any confusion. Further, we can put this information (test_size) into a variable and use it to populate learning_stages which could help the library users to better integrate their scripts.